client/app/scripts/superdesk-publish/publish.js

```diff
@@ -283,7 +283,13 @@
             );
         };
 
-        $scope.filterSchedule = function() {
+        $scope.filterSchedule = function(item, type) {
+            if (type === 'channel') {
+                $scope.selectedFilterChannel = item;
+            } else if (type === 'subscriber') {
+                $scope.selectedFilterSubscriber = item;
+            }
```

Comment:
Test for this function exist, difference is passing parameters, because in earlier version there was select with options, and now I changed it to custom dropdown and I needed somehow to pass values. This little change just assign values to predefined variables.